### PR TITLE
rebasing and squashing wabrez/integrat-vott-react

### PR DESCRIPTION
**Pull request by luisamiranda**

### CODE REVIEW

**Comment by wbreza** [comment]
@luisamiranda - Looks like you have some compilation issues.

**Comment by luisamiranda** [comment]
The compilation issues are all in files not changed by this PR: 
src/react/components/pages/editorPage/canvas.test.tsx(61,35): error TS2554: Expected 3 arguments, but got 2.
src/react/components/pages/editorPage/canvas.test.tsx(147,23): error TS2339: Property 'onRegionMoveEnd' does not exist on type 'Editor'.
src/react/components/pages/editorPage/canvas.tsx(51,21): error TS2339: Property 'onRegionMoveEnd' does not exist on type 'Editor'.
src/react/components/pages/editorPage/editorPage.test.tsx(84,35): error TS2554: Expected 3 arguments, but got 2.

File: package.json [hunk 1]
```diff
@@ -29,7 +31,8 @@
         "rimraf": "^2.6.2",
         "shortid": "^2.2.14",
         "video-react": "^0.13.2",
-        "vott-ct": "^2.1.17"
+        "vott-ct": "^2.1.11",
```
Comment (PIC123) [inline package.json]: This shouldn't downgrade the canvas tools library. This could be the cause of the compilation errors. Npm install should fix this

Comment (luisamiranda) [reply package.json]: Thank you!

**Pull request merged**